pymedphys/_experimental/pseudonymisation/__init__.py

```diff
@@ -43,9 +43,18 @@ def get_default_identifying_uids():
 
 @functools.lru_cache()
 def _get_default_pseudonymisation_keywords():
-    anon_keyword_set = set(get_default_identifying_keywords())
-    psuedo_uid_set = set(get_default_identifying_uids())
-    return tuple(anon_keyword_set.union(psuedo_uid_set))
```

Comment:
Silly me, I missed that this was explicitly only psudonymisation. A misread from flying through too quick.